jkube-kit/common/src/test/java/org/eclipse/jkube/kit/common/util/EnvUtilTest.java

```diff
@@ -62,4 +197,12 @@ public void testLoadTimestampShouldLoadFromFile() throws Exception {
         // Then
         assertThat(timestamp, equalTo(new Date(1445385600000L)));
     }
+
+    @Test
+    public  void testIsWindows(){
+        //When
+        boolean result = isWindows();
+        //Then
+        assertEquals(false,result);
```

Comment:
I think this can be simplified to `assertFalse(isWindows())`

Reply:
Understood!